.circleci/config.yml

```
@@ -47,6 +47,7 @@ jobs:
                 --ignore-url 'https://dicomlookup.com/dicomtags/.*' \
                 --ignore-url 'https://www.instagram.com/bidsstandard/' \
                 --ignore-url 'https://jsoneditoronline.org' \
+                --ignore-url 'https://rrid.site.*' \
```

To include the top level site

```suggestion
                --ignore-url 'https://rrid.site.*' \
```

---

Never mind, looks like you got there first.